pkgs/servers/immich/default.nix

```diff
@@ -0,0 +1,238 @@
+{ lib
+, pkgs
+, buildNpmPackage
+, fetchFromGitHub
+, fetchPypi
+, python
+, nodejs
+, nixosTests
+# build-time deps
+, pkg-config
+, makeWrapper
+, cmake
+# runtime deps
+, ffmpeg
+, imagemagick
+, libraw
+, vips
+}:
+let
+  buildNpmPackage' = buildNpmPackage.override { inherit nodejs; };
+  sources = lib.importJSON ./sources.json;
+  inherit (sources) version;
+
+  src = fetchFromGitHub {
+    owner = "immich-app";
+    repo = "immich";
+    rev = "v${version}";
+    inherit (sources) hash;
+  };
+
+  cli = buildNpmPackage' {
```

Comment:
There is already a package `immich-cli`.

Reply:
Yes but here we have the original source and the existing immich-cli just pulls it from npm. Either they should live on as separate packages like they do now, or the immich-cli should be removed.

Reply:
No it was repackaged to use `buildNpmPackage` in https://github.com/NixOS/nixpkgs/pull/250280.

Reply:
Upstream should add a remark about that to the former repo's readme. Maybe you wanna make a PR?